src/XMakeTasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -1605,6 +1633,17 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   <!--
     ============================================================
+                                         IsCrossTargetingProject
+
+    Overrridden by cross-targeting projects to return "true" if
+    the project should have its GetTargetFrameworkProperties
+    target called with a ReferringTargetFramework. If "false",
+    the single output of the project is used.
+  -->
+  <Target Name="IsCrossTargetingProject" Returns="false" />
```

Comment:
Does the Returns="true" version need to go in the SDK or should it be in Microsoft.Common.CrossTargeting.targets?